TLM/TLM/Custom/AI/CustomTramBaseAI.cs

```diff
@@ -6,77 +6,106 @@
     using ColossalFramework;
     using ColossalFramework.Math;
     using CSUtil.Commons;
-    using Custom.PathFinding;
+    using JetBrains.Annotations;
     using Manager;
     using Manager.Impl;
+    using PathFinding;
     using RedirectionFramework.Attributes;
     using State;
+    using State.ConfigData;
     using UnityEngine;
 
+    // TODO inherit from VehicleAI (in order to keep the correct references to `base`)
     [TargetType(typeof(TramBaseAI))]
-    public class CustomTramBaseAI : TramBaseAI { // TODO inherit from VehicleAI (in order to keep the correct references to `base`)
```

Comment:
comment is missing, but still valid

Reply:
It is 2 lines above

![image](https://user-images.githubusercontent.com/288724/61579528-6ab65b00-ab06-11e9-86ee-dd6111fc8438.png)